lca_activity_browser/app/ui/tables/inventory.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+from PyQt5 import QtCore, QtGui, QtWidgets
+import itertools
+
+from .table import ABTableWidget, ABTableItem
+from ..icons import icons
+from ...signals import signals
+from ...bw2extensions.commontasks import *
```

Comment:
Could you maybe add explicit imports here? Does pycharm not complain about wildcard imports?